src/app/ssmt/ssmt-report/report-graphs/ssmt-pie-chart/ssmt-pie-chart.component.html

```diff
@@ -1,4 +1,11 @@
-<div #ssmtPieChart></div>
+<div #ssmtPieChart [ngStyle]="{'display': ssmt.valid.isValid? 'block' : 'none'}"></div>
 <div class="w-100 text-center" *ngIf="noData">
```

Comment:
Our check for noData is if the turbines generate energy. They are showing up for invalid as well. Should only show up if valid and no generation